docs/source/editor/prepub.rst

```diff
@@ -0,0 +1,99 @@
+
+.. warning::
+   **If an author requests a licence other than CC BY-4.0, this must be flagged in advance to the typesetters.** 
+
+
+Once you have thoroughly checked the metadata, check the ‘Mark as Complete’ box to move onto the next step. 
+
+2. Set Issue
+------------
+
+Set Issue, the second step in Pre Publication, is where you can assign the article’s issue(s), if this has not been done already. 
+Click ‘Add to Issue’ to add to an existing issue (you can select more than one issue if needed, for example, if the article is part of a special issue as well as a journal volume) or ‘Create an issue’ at the bottom of the pop up window if the article belongs to a new issue. Then click the X at the top of the pop up box to close.
+
+Check the ‘Mark as Complete’ box to move onto the next step. 
+
+3. Verify DOIs
+--------------
+
+Articles usually have a DOI, or Digital Object Identifier, automatically assigned and ‘minted’ (created) for them by the system. This unique digital link to the article is usually created automatically for you by the system, and the Verify DOI stage lets you check if the DOI has been issued successfully or if there are any problems.
+If the DOI only has two ‘hops’, or two steps shown as bullet points in the Verify DOI stage, this means that no further changes are needed and you are ready to proceed. 
+You can verify that the article’s DOI is resolving correctly by clicking ‘Edit’ in the top right hand corner of the pop up window and checking that the ‘Status’ column reads ‘Success’. If the DOI Status is listed as pending, or blank, click ‘Poll for DOI Status’. If no DOI has been assigned, click ‘Issue DOI’. NB that newly issued DOIS can take a while to go through.  
+
+
+.. figure:: ../nstatic/prepubgif2.gif
+
+Check the ‘Mark as Complete’ box to move onto the next step. 
+
+4. Select a Galley for Rendering
+--------------------------------
+
+For OLH journals publishing on Janeway, always select the XML galley for rendering. 
+**Check to make sure no additional files (beside the XML and PDF) are visible here.** If they are, they will also be published alongside the proofed galleys. If any other files appear here, other than one XML and one PDF file, return to the article page to delete any other files that have accidentally been added as Galleys before proceeding.
+
+
+.. figure:: ../nstatic/prepubgif3.gif
+
+
+.. warning::
+   **If you have not done so already, double check (in the Typesetting Plugin) that any typesetting queries at the very end of the PDF have been addressed and removed.** 
+
+
+Check the ‘Mark as Complete’ box to move onto the next step. 
+
+5. Set a Publication Date 
+-------------------------
+
+Select a date and time for publication. Selecting today's date will make it available immediately. A date for publication must be set for the piece to publish. 
```

Comment:
Perhaps should mention that the dates/times are related to Profile timezone?

Reply:
Will do

Reply:
If no timezone is selected, the default is GMT right?

Reply:
UTC